### PR TITLE
Add schema to examples and fix formatting

### LINKED ISSUE #3748: Add corresponding schema to 'Seeding your database' page
The [Seeding your database](https://www.prisma.io/docs/guides/database/seed-database) page has an example seeding script, but doesn't have a schema to go with it - should add one in to make it easier to follow.

### PR DESCRIPTION
**Pull request by keerlu**
## Describe this PR

- Added corresponding Prisma schema to 'Seed your database' example to make it easier to follow
- I took the seed code out of 'expand to view' sections because I was failing to spot it at all on a first look and getting confused
- some formatting tweaks

## What issue does this fix?

Fixes #3748

### CODE REVIEW

File: content/300-guides/050-database/300-seed-database.mdx [hunk 1]
```diff
@@ -305,3 +350,4 @@ psql file.sql
 </tab>
 
 </TabbedContent>
+````
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]:
```suggestion
```

File: content/300-guides/050-database/300-seed-database.mdx [hunk 2]
```diff
@@ -76,71 +76,92 @@ Here we suggest some specific seed scripts for different situations. You are fre
...
+   ```prisma file=schema.prisma
+   generator client {
+     provider = "prisma-client-js"
+   }
+   
+   datasource db {
+     provider = "postgresql"
+     url      = env("DATABASE_URL")
+   }
+
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]: If you want to keep the schema shorter, you could leave this off I think.

File: content/300-guides/050-database/300-seed-database.mdx [hunk 3]
```diff
@@ -166,13 +187,13 @@ Here we suggest some specific seed scripts for different situations. You are fre
 
    Some projects may require you to add compile options. When using Next.js for example, you would setup your seed script like so:
 
-   ```ts
+   ```json file=package.json
    "prisma": {
-      "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
+   "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]:
```suggestion
     "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
```

File: content/300-guides/050-database/300-seed-database.mdx [hunk 4]
```diff
@@ -166,13 +187,13 @@ Here we suggest some specific seed scripts for different situations. You are fre
...
    },
    ```
 
-5. To seed the database, run the `db seed` CLI command:
+   5. To seed the database, run the `db seed` CLI command:
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]:
```suggestion
5. To seed the database, run the `db seed` CLI command:
```

File: content/300-guides/050-database/300-seed-database.mdx [hunk 5]
```diff
@@ -258,7 +303,7 @@ Here we suggest some specific seed scripts for different situations. You are fre
    }
    ```
 
-4. To seed the database, run the `db seed` CLI command:
+   3. To seed the database, run the `db seed` CLI command:
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]:
```suggestion
4. To seed the database, run the `db seed` CLI command:
```

File: content/300-guides/050-database/300-seed-database.mdx (hunk 2 above)
Comment (keerlu) [reply content/300-guides/050-database/300-seed-database.mdx]: Thanks, removed

**Review by garrensmith — APPROVED** [review]
Looks good.

**Review by janpio — CHANGES_REQUESTED** [review]
Javascript tab: 
image

**Comment by keerlu** [comment]
> Javascript tab: image

@janpio thanks, should be fixed now

**Review by janpio — APPROVED** [review]

**Pull request merged**